### DISCUSSION TITLE
Newsticker Widget

### DISCUSSION THREAD

**Discussion by ghost** [templaza/astroid-framework#549] [Ideas]
Sometimes a news ticker on the website can be very useful. For example for club websites. A news ticker widget would therefore also be quite good. Maybe something like this.

[Newsticker Demo](https://jdemo.bestaddon.com/extensions/ba-newsticker/)

**Comment by Chacapamac** [comment]
Do you think it should work with with chosen article category(ies) of article  (e.g. an hypothetical “blog” or news category (or both) and display the title and/or image of the articles in the  category(ies) choosen?

**Comment by ghost** [comment]
In terms of design, it should perhaps have similar options as in the linked demo. Selecting the items displayed by item category or item ID.
